resources/lib/services/playback/action_controller.py

```diff
@@ -149,18 +152,23 @@ def _get_player_state(self, player_id=None):
         except IOError:
             return {}
 
-        # Sometime may happen that when you stop playback, a player status without data is read,
-        # so all dict values are returned with a default empty value,
-        # then return an empty status instead of fake data
-        if not player_state['audiostreams']:
-            return {}
-
         # convert time dict to elapsed seconds
         player_state['elapsed_seconds'] = (
             player_state['time']['hours'] * 3600 +
             player_state['time']['minutes'] * 60 +
             player_state['time']['seconds'])
 
+        # Sometime may happen that when you stop playback, a player status without data is read,
+        # or while Kodi is stopping video it will return time as 0
+        if (player_state['audiostreams'] and player_state['elapsed_seconds']) or \
+           (player_state['audiostreams'] and not player_state['elapsed_seconds'] and not self._last_player_state):
```

Comment:
to split if condition, enclose all if condition inside parenthesis (..condition..) and remove slash